tools/kfdrc_peddy_tool.cwl

```diff
@@ -16,7 +16,7 @@ arguments:
       -m peddy
       -p 4
       --sites hg38
-      --prefix $(inputs.output_basename)
+      --prefix $(inputs.output_basename).peddy
```

Comment:
hmmm, can see why you added, but then the peddy  output becomes peddy.peddy...I'm of two minds on this one

Reply:
Git blame says you did it: https://github.com/kids-first/kf-jointgenotyping-workflow/commit/84d03071a71c363898594597490b97ed6db7d017

Reply:
Someone should fire that guy! I'll agree with past self and let it be.